Maestro/subscriptions.json

```diff
@@ -1061,10 +1059,10 @@
         }
       }
     },
-    // Automate opening PRs to merge extensions release/5.0-previewN branches back to master.
+    // Automate opening PRs to merge ef6 release/6.4 branch back to master.
     {
       "triggerPaths": [
-        "https://github.com/dotnet/extensions/blob/release/5.0//**/*"
+        "https://github.com/dotnet/ef6/blob/release/6.4//**/*"
```

Comment:
nit: Why not put this into the wildcard / aspnetcore-tooling 3.1 merge to master rule❔